patterns/caching-static-data-when-transferring.md

```diff
@@ -0,0 +1,42 @@
+# Caching static data instead of transferring it
+
+## Version
+1.0
+
+## Submitted By
+Sarah Hsu
+
+## Published Date
+TBD
+
+## Intent
+Caching static data instead of transferring ot
+
+## Tags
+Cloud, Networking, Cloud Engineer, Small
+
+## Problem
```

Comment:
Guessing this should be something like 
"From an energy-efficiency perspective, it's better to reduce network traffic by reading the data locally than by accessing it remotely over the network."